CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Added definition for `heartbeat.polling.interval` for CDN Traffic Monitor config in API documentation.
 - New `pkg` script options, `-h`, `-s`, `-S`, and `-L`.
 - Added `Invalidation Type` (REFRESH or REFETCH) for invalidating content to Traffic Portal.
+- Added a new field `geoLimitCountriesList` to the `DeliveryService` structure to accept both an array as well as single strings for geo limit country codes
```

Comment:
This is no longer true - also currently

> an array as well as single strings

isn't true currently, see above comment